integration/terraform/ec2/linux/README.md

```diff
@@ -122,4 +122,18 @@ This process generates a new ami we can then use for testing
    2. Go to local stack directory
       1. cd ${path to agent dir}/integration/terraform/ec2/localstack
    3. Tear down localstack state
-      1. terraform destroy --auto-approve
\ No newline at end of file
+      1. terraform destroy --auto-approve
+
+**How To Run On Your Own Fork**
+1. Follow "Create resources and setup local" except install terraform
+   1. You may skip installing terraform since terraform will be installed on GitHub action runners
+2. Set up GitHub action secrets in your fork
+   1.   ```
+        PRIVATE_KEY: ${Your private key}
+        TERRAFORM_AWS_ACCESS_KEY_ID: ${User aws access key}
+        TERRAFORM_AWS_SECRET_ACCESS_KEY: ${User aws secret key}
+        S3_INTEGRATION_BUCKET: ${Bucket to save build}
+        KEY_NAME: ${Key pair name for ec2}
+        VPC_SECURITY_GROUPS_IDS: ${Security group within your vpc}
+        IAM_ROLE: ${Role the ec2 instance should assume}
+        ```
```

Comment:
I had to search how to do this.
It may be more helpful to mention this is the URL:
https://github.com/<github_username>/amazon-cloudwatch-agent/settings/secrets/actions

Do the values need to be surrounded in double quotes? Escaped?
For example should `KEY_NAME` have a value `"abc123"` or just `abc123`... Maybe both work...

Reply:
Do not add quotes. Only places quotes should be is ["sg-05d0fe0bbc64c8133"]

Reply:
Added more to the doc